airflow/Dockerfile

```diff
@@ -77,11 +77,13 @@ LABEL name="Apache Airflow" \
 COPY airflow/licenses /licenses
 
 # Update image and install python
+# g++ is required for libstdc++ that is preloaded in entrypoint.sh
 RUN microdnf update && \
-    microdnf install  \
+    microdnf install -y \
```

Comment:
I think we don't need the `-y` here since we have `assumeyes=True` in `dnf.conf`

Reply:
deleted again https://github.com/stackabletech/docker-images/pull/778/commits/d6c7fc55b1e94b519d4f3d0d87939525228a826d